### ISSUE TITLE
Alt+space is unable to consume event.

### ISSUE THREAD

**Issue by zgqq** [tulskiy/jkeymaster#29]
Firefox still will response whether on Win7 or Win10 when alt+space is registered as hotkey.

**Comment by tulskiy** [comment]
If it's an os hotkey or already registered by another application, then there's nothing I can do. Global hotkeys are exclusive to whoever registered it, that's how the api works. What you need is a global keyboard hook like https://github.com/kwhat/jnativehook

**Comment by zgqq** [comment]
@tulskiy No,you didn't understand what i said. Firefox will show its menu when merely pressing alt,so it still response even if alt+space is registered as hotkey successfully.If you press alt+space and don't release until own dialog pops,firefox does't show its menu.Namely,hotkey event is received too late.
